tests/caasp/first_boot.pm

```diff
@@ -26,42 +26,38 @@ sub run {
     assert_screen 'linux-login-casp', 300;
 
     # Workers installed using autoyast have no password - bsc#1030876
-    unless (get_var('AUTOYAST')) {
-        # Workaround for bsc#1035968
-        if (is_caasp 'VMX') {
-            # FreeRDP is not sending 'Ctrl' as part of 'Ctrl-Alt-Fx', 'Alt-Fx' is fine though.
-            my $ctrl = check_var('VIRSH_VMM_FAMILY', 'hyperv') ? '' : 'ctrl-';
-            my $tty2 = wait_screen_change(sub { send_key "${ctrl}alt-f2"; }, 1);
-            unless ($tty2) {
-                wait_screen_change(undef, 180);
-                wait_still_screen;
-            }
-        }
-        select_console 'root-console';
+    return if get_var('AUTOYAST');
 
-        # On Hyper-V we need to add special framebuffer provisions
-        if (check_var('VIRSH_VMM_FAMILY', 'hyperv') || check_var('VIRSH_VMM_TYPE', 'linux')) {
-            set_framebuffer_resolution;
-            assert_script_run 'transactional-update grub.cfg';
-            process_reboot 1;
-        }
+    # Workaround for bsc#1035968
```

Comment:
if it is workaround , why you not add recordsoftfailure ? Also I check bug that you mention in comment and it's state is "RESOLVED FIXED" - probably it need to be reopened ?

Reply:
It's a *comment* that's still valid and was there before (line 30) ...just for reference...

Reply:
I have read bug , first of all it was closed with wrong reason in a fact it is "RESOLVED WONT FIX" which change a lot. What about replacing this line with 
"Login is displayed before cloud-init finishes" ?

Reply:
Fixed, I also left reference to bug